engine/src/flutter/shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -1649,6 +1649,100 @@ TEST_F(EmbedderTest, CanRemoveView) {
   ASSERT_EQ(message, "View IDs: [0]");
 }
 
+TEST_F(EmbedderTest, RemoveViewCallbackIsInvokedAfterRasterThreadIsDone) {
```

Comment:
Consider adding a link to make it easier to find the context for this bug:

```suggestion
// Regression test for:
// https://github.com/flutter/flutter/issues/164564
TEST_F(EmbedderTest, RemoveViewCallbackIsInvokedAfterRasterThreadIsDone) {
```